app/mailers/submission_mailer.rb

```diff
@@ -1,6 +1,13 @@
 class SubmissionMailer < ApplicationMailer
+
+  def send_contract(submission)
+    @submission = submission
+    attachments["Umowa-wydawnicza.pdf"] = File.read("#{Rails.root}/public/plik.pdf")
+    mail(to: @submission.person.email, subject: 'Umowa wydawnicza')
```

Comment:
Email powinien być wysyłany do wszystkich autorów. `@submission.person.email` to majl redaktora.
Żeby to zadziałało jak należy, trzeba do metody wysyłającej mejle dodać parametr `person`, który wskazuje na osobę do której mejl jest wysyłany - i to na jej podstawie określić email oraz nazwisko.